cmd/scollector/collectors/redis_unix.go

```diff
@@ -182,13 +182,19 @@ func c_redis() (opentsdb.MultiDataPoint, error) {
 			continue
 		}
 		defer c.Close()
-		lines, err := c.Do("INFO")
-		if err != nil {
-			Error = err
+		info_command, info_command_err := c.Do("info", "commandstats")
```

Comment:
can you move the call to `c.Do("info", "commandstats")` down to right above where you iterate over the results?